### PR TITLE
shop poster notification

### PR DESCRIPTION
**Pull request by nicosampler**

### CODE REVIEW

File: pages_/my-shop.tsx [hunk 1]
```diff
@@ -117,6 +131,16 @@ const MyShop = ({ isLoggedIn, shopId }: Props) => {
     }
   }, [isLoggedIn, shopId]);
 
+  useEffect(() => {
+    const res = window.localStorage.getItem(POSTER_NOTIFICATION);
+
+    if (res) {
```
Comment (ramirotw) [inline pages_/my-shop.tsx:137]: shouldn't this be `res === 'true'` just to be sure?

File: src/components/Notification/index.tsx [hunk 2]
```diff
@@ -57,23 +59,34 @@ export default function Notification({
...
       <Modal.Body className="py-5 px-3 text-center">
         <p>
           {subTitle && <strong className="d-block mb-3">{subTitle}</strong>}
           <span className="text-muted">{message}</span>
         </p>
+      </Modal.Body>
+      <Modal.Footer>
+        {onCancel && (
+          <Button
+            size={'lg'}
```
Comment (ramirotw) [inline src/components/Notification/index.tsx:76]: don't you like `size="lg"` more?

File: src/components/Notification/index.tsx [hunk 3]
```diff
@@ -57,23 +59,34 @@ export default function Notification({
...
+            onClick={onCancel}
+            className="text-dark"
+          >
+            {t('common:close')}
+          </Button>
+        )}
         <Button
           variant="primary"
-          size="lg"
+          size={'lg'}
```
Comment (ramirotw) [inline src/components/Notification/index.tsx:86]: :(

File: pages_/my-shop.tsx (hunk 1 above)
Comment (nicosampler) [reply pages_/my-shop.tsx:137]: it shouldn't be set with another value, but adding === 'true' won't hurt anybody :)

**Pull request merged**